### PR TITLE
fix: Reduced borderWidth for InspectionSymbol, and removed to-from zone names

### PR DESCRIPTION
**Pull request by strandlie**
Closes AtB-AS/kundevendt#19517

### CODE REVIEW

**Review by reidzeibel — COMMENTED** [review]
I think the removal of zone name should be followed by the design update for the ticket stuffs, because if we remove it now, the user has no way to know the valid zone of their ticket. 

We need the zone indicator to be added before we remove the zone name.
image

**Comment by strandlie** [comment]
> I think the removal of zone name should be followed by the design update for the ticket stuffs, because if we remove it now, the user has no way to know the valid zone of their ticket.
> 
> We need the zone indicator to be added before we remove the zone name. image

I will update all designs included in AtB-AS/kundevendt#4220 before I merge this. Hang tight for a couple more PRs coming soon :)

**Comment by gorandalum** [comment]
> I think the removal of zone name should be followed by the design update for the ticket stuffs, because if we remove it now, the user has no way to know the valid zone of their ticket.
> 
> We need the zone indicator to be added before we remove the zone name. image

Just a note that we do show zone names on fare contracts as text as well, not just as part of the inspection symbol.
Screenshot 2024-12-05 at 10 53 46

File: src/fare-contracts/components/InspectionSymbol.tsx [hunk 1]
```diff
@@ -27,8 +26,6 @@ export type InspectionSymbolProps = {
 
 export const InspectionSymbol = ({
   preassignedFareProduct,
-  fromTariffZone,
-  toTariffZone,
```
Comment (gorandalum) [inline src/fare-contracts/components/InspectionSymbol.tsx:31]: Can also remove these from the `InspectionSymbolProps`

Comment (strandlie) [reply src/fare-contracts/components/InspectionSymbol.tsx:31]: Done

**Comment by strandlie** [comment]
> > I think the removal of zone name should be followed by the design update for the ticket stuffs, because if we remove it now, the user has no way to know the valid zone of their ticket.
> > We need the zone indicator to be added before we remove the zone name. image
> 
> I will update all designs included in [AtB-AS/kundevendt#4220](https://github.com/AtB-AS/kundevendt/issues/4220) before I merge this. Hang tight for a couple more PRs coming soon :)

Not true any more. Will merge this now, as the mentioned change is very nearly finished.

**Pull request merged**